modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -58,16 +57,38 @@
 import java.util.concurrent.ExecutorService;
 
 /**
- * Input attributes to log is (iijTimeStamp (Long), value (Float)).
+ * Input attributes to log is (iijTimeStamp (Long), type (String) , Count(Integer) ,
+ * Windowsize (Integer) , id(String)).
  */
 @Extension(
         name = "throughput",
         namespace = "throughput",
         description = "Measuring performance of stream processor with simple passthrough",
         parameters = {
                 @Parameter(name = "iijtimestamp",
-                           description = "This value used to find the sending timestamp from client",
-                           type = {DataType.LONG}),
+                        description = "This value used to find the sending timestamp from client",
+                        type = {DataType.LONG}),
+
+                @Parameter(name = "type",
+                        description = "This value used to specify which metric value " +
+                                "you are going to exract(\"throughput\" " +
+                                "or \"latency\" or \"both\" )",
+                        type = {DataType.STRING}),
+
+                @Parameter(name = "windowsize",
```

Comment:
change it to : window.size

Reply:
Fixed https://github.com/wso2/product-sp/pull/880/commits/9ee8cbb4f4af6e57527b1f8b83363f2fc400a552